dashboard/db/migrate/20160715015742_set_type_for_pd_workshop_sections.rb

```diff
@@ -0,0 +1,13 @@
+class SetTypeForPdWorkshopSections < ActiveRecord::Migration
+  def up
+    Pd::Workshop.where.not(section:nil).each do |workshop|
+      workshop.section.update!(type: Section::TYPE_PD_WORKSHOP)
```

Comment:
Doh! You're right. This was leftover from before I updated the column name. Fixed.